bin/install-qa-check.d/80multilib-strict

```diff
@@ -6,14 +6,13 @@ multilib_strict_check() {
 	then
 		rm -f "${T}/multilib-strict.log"
 		local abort=no dir file
-		MULTILIB_STRICT_EXEMPT=$(echo ${MULTILIB_STRICT_EXEMPT} | sed -e 's:\([(|)]\):\\\1:g')
 		for dir in ${MULTILIB_STRICT_DIRS} ; do
 			[[ -d ${ED}/${dir} ]] || continue
-			for file in $(find ${ED}/${dir} -type f | grep -v "^${ED}/${dir}/${MULTILIB_STRICT_EXEMPT}"); do
-				if file ${file} | egrep -q "${MULTILIB_STRICT_DENY}" ; then
+			while read -r -d '' file; do
+				if file "${file}" | egrep -q "${MULTILIB_STRICT_DENY}" ; then
```

Comment:
shellcheck suggests: SC2196: egrep is non-standard and deprecated. Use grep -E instead.